### PR TITLE
Infra Integration Demo

### PR DESCRIPTION
**Pull request by l-monninger**
# Summary
- **Status**: `draft` 
- **RFCs**: [Link to RFC](#./link/to/rfc), [Link to RFC](#./link/to/rfc), or $\emptyset$.
- **Categories**: any of `protocol-units`, `networks`, `scripts`, `util`, `cicd`, or `misc`.

# Changelog

# Testing

# Outstanding issues

### CODE REVIEW

**Comment by l-monninger** [comment]
gali: I want some infra for monza

**Comment by l-monninger** [comment]
**Gali PR:** https://github.com/movementlabsxyz/infra/pull/32

**Comment by 0xmovses** [comment]
Thanks for this. Is this ready for review? Could you write in the PR description some notes as in `PR.md` and let me know how I should be testing this.

File: .github/actions/gali/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# `gali`
```
Comment (mzabaluev) [inline .github/actions/gali/README.md:1]: Cool stuff, we want to back our processes with automation.

Is this meant to be incubation for a `gali` action that will be hosted in its own repository some day?

**Pull request merged**